src/fparser/tests/test_api.py

```diff
@@ -97,7 +116,12 @@ def test_dimension_attr():
     assert bvar.dimension == [('-20', '20'), ('100', '113'), ('-  512', '713')]
     assert bvar.shape == ['41', '14', '1226']
 
+
 def test_provides():
+    '''
+    Tests for non-Fortran (pyf?) module_provides and use_provides
```

Comment:
It's because I'm none too sure what the "non-Fortran" means but I'm guessing it's the "pyf" format (whatever that is).

Reply:
[This PYF file format description link](https://datatypes.net/open-pyf-files) describes "pyf" as a F2PY Signature File, so I think the question mark can be removed. Perhaps it would be worth adding a short description of what pyf is?

Reply:
Good point - I've elaborated!